doc/en/multitest.rst

```diff
@@ -276,6 +276,40 @@ and overriding ``get_output`` method.
 An example implementation of custom test lister can be seen
 :ref:`here <example_multitest_listing_custom>`.
 
+If the lister provide, name and description it can be used to extend the ``--info`` commandline
+parameter and the lister can be selected from the commandline.
```

Comment:
This sentence is a little tricky to understand. Can you improve the wording?
"If the lister provide, ... "

Reply:
is this looking better?

> Listers can be registered to be used with the ``--info`` commandline parameter the same way as the built in listers.
> 
> The custom lister class should provide: